styles/crunchyroll/catppuccin.user.css

```diff
@@ -0,0 +1,2501 @@
+/* ==UserStyle==
+@name Crunchyroll Catppuccin
+@namespace github.com/catppuccin/userstyles/styles/crunchyroll
+@homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/crunchyroll
+@version 0.0.1
+@updateURL https://github.com/catppuccin/userstyles/raw/main/styles/crunchyroll/catppuccin.user.css
+@supportURL https://github.com/catppuccin/userstyles/issues?q=is%3Aopen+is%3Aissue+label%3Acrunchyroll
+@description Soothing pastel theme for Crunchyroll
+@author Catppuccin
+@license MIT
+
+@preprocessor less
+@var select lightFlavor "Light Flavor" ["latte:Latte*", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha"]
+@var select darkFlavor "Dark Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve*", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+@var select playerFlavor "Player: Flavor" ["auto:Auto*", "latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha"]
+@var select playerSetting "Player: Setting" ["none:None", "minimal:Minimal", "medium:Medium*", "full:Full"]
```

Comment:
Can you elaborate more on these player options; why a user might want them enabled/disabled compared to other users (in other words, why is this an option), and what do each of these options look like?

Reply:
See https://github.com/catppuccin/userstyles/pull/1023#issuecomment-2344429823